pkg/db/db.go

```diff
@@ -625,20 +625,18 @@ type GetLeasesOutput struct {
 
 // GetLeases takes a set of filtering criteria and scans the Leases table for the matching records.
 func (db *DB) GetLeases(input GetLeasesInput) (GetLeasesOutput, error) {
-	limit := int64(25)
 	filters := make([]string, 0)
 	filterValues := make(map[string]*dynamodb.AttributeValue)
 
-	if input.Limit > 0 {
-		limit = input.Limit
-	}
-
 	scanInput := &dynamodb.ScanInput{
 		TableName:      aws.String(db.LeaseTableName),
-		Limit:          &limit,
 		ConsistentRead: aws.Bool(db.ConsistentRead),
 	}
 
+	if input.Limit > 0 {
+		scanInput.Limit = &input.limit
```

Comment:
Actually this doesn't look like this works with the reference

Reply:
Oops had a letter casing error haha... The build is successful now. @dilip0515 is also getting me access to the devops